compiler/src/openxla/compiler/nvgpu/Dialect/CUDNN/Conversion/ConvertCUDNNToRuntime.cpp

```diff
@@ -663,6 +683,39 @@ struct ConvertCudnnBiasOp : public CudnnOpConversionPattern<BiasOp> {
   }
 };
 
+//===----------------------------------------------------------------------===//
+// cudnn.relu
+//===----------------------------------------------------------------------===//
+
+struct ConvertCudnnReluOp : public CudnnOpConversionPattern<ReluOp> {
+  using CudnnOpConversionPattern::CudnnOpConversionPattern;
+
+  LogicalResult matchAndRewrite(
+      ReluOp op, OpAdaptor adaptor,
+      ConversionPatternRewriter &rewriter) const override {
+    MLIRContext *ctx = rewriter.getContext();
+    ImplicitLocOpBuilder b(op->getLoc(), rewriter);
+    SmallVector<Value> args(adaptor.getOperands());
+    auto f32 = rewriter.getF32Type();
+
+    // It is unable to convert the lowerClip directly to float, thus the static
+    // cast.
+    float lower_clip =
+        static_cast<float>(adaptor.getLowerClip().convertToDouble());
```

Comment:
Do you need static_cast here? double should be assignable to float

Reply:
I vaguely remember it giving me a warning, which was annoying.

Reply:
Removed the static cast.  A while back I vaguely remember it giving me some warning. Looks like it is not happening anymore , so I took it off.